<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Vault Explorer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
	background-color: rgb(0, 0, 0);}</string>
  </property>
  <widget class="QPushButton" name="select">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>255</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#select{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#select{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="dirName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Root&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>205</y>
     <width>106</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Write to Directory:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>20</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Go to Root</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#home{
image:url(:/nSOTScanner/Pictures/homeIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#home{
image:url(:/nSOTScanner/Pictures/homeIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Directories">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>89</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#838383;&quot;&gt;Directories:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="dirList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget#dirList{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}


QScrollBar:horizontal {
   border: 0px;
   background: black;
   height: 15px;
}
QScrollBar:vertical {
   border: 0px;
   background: black;
   width: 15px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="DataSets">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>77</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#838383;&quot;&gt;Data Sets:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentDir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#currentDir{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>25</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;üè†&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Back</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#back{
image:url(:/nSOTScanner/Pictures/backIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#back{
image:url(:/nSOTScanner/Pictures/backIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#fileList{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}


QScrollBar:horizontal {
   border: 0px;
   background: black;
   height: 15px;
}
QScrollBar:vertical {
   border: 0px;
   background: black;
   width: 15px;
}</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>205</y>
     <width>106</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Open Data Set:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>225</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#currentFile{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelSelect">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>255</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#cancelSelect{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#cancelSelect{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addDir">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add Directory</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#addDir{
image:url(:/nSOTScanner/Pictures/addPlotIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#addDir{
image:url(:/nSOTScanner/Pictures/addPlotIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dvexplorer_refresh">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>20</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_dvexplorer_refresh{
image:url(:/nSOTScanner/Pictures/refreshIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#refresh{
image:url(:/nSOTScanner/Pictures/refreshIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
